non_profit/non_profit/doctype/donation/donation.py

```diff
@@ -65,6 +65,7 @@ def create_payment_entry(self, date=None):
 		pe.flags.ignore_mandatory = True
 		pe.insert()
 		pe.submit()
+		self.db_set("paid", 1)
```

Comment:
We can move this to line number 43 so that donation gets set as Paid even if payment entry creation is not enabled because the payment was successful anyway.

```python
def on_payment_authorized(self, *args, **kwargs):
	  self.db_set("paid", 1)
	  self.load_from_db()
	  self.create_payment_entry()
```